lib/rspec/openapi/record_builder.rb

```diff
@@ -35,4 +35,14 @@ def find_route(request)
     end
     raise "No route matched for #{request.request_method} #{request.path_info}"
   end
+
+  # workaround to get real request parameters
+  # because ActionController::ParamsWrapper overwrites request_parameters
+  def raw_request_params(request)
```

Comment:
I've found a workaround to get real request params but it's not beautiful. Any idea?

Reply:
I'm not sure if I understand the background. Could you add a request spec under spec/requests which fails without the workaround? Currently just using `request.request_parameters` generates the same doc.

Reply:
Oh, I just forgot using `OPENAPI=1`. I'll take a look at what's going on this night.